clients/githubrepo/contributors_e2e_test.go

```diff
@@ -24,11 +25,18 @@ import (
 
 	"github.com/ossf/scorecard/v5/clients"
 	"github.com/ossf/scorecard/v5/clients/githubrepo/roundtripper"
+	"github.com/ossf/scorecard/v5/internal/gitfile"
 	"github.com/ossf/scorecard/v5/log"
 )
 
 var _ = Describe("E2E TEST: githubrepo.contributorsHandler", func() {
 	var contribHandler *contributorsHandler
+	var fileHandler io.ReadCloser
+	repoURL := Repo{
+		owner:     "ossf",
+		repo:      "scorecard",
```

Comment:
```go
		owner:     "ossf-tests",
		repo:      "scorecard-check-contributors-e2e",
```

Reply:
And can you delete the other repoURL definition below?
https://github.com/ossf/scorecard/blob/c036e71edd2ed631864f7437574eba7720d52b4b/clients/githubrepo/contributors_e2e_test.go#L68-L72